### ISSUE TITLE
Full API reference link is down.

### ISSUE THREAD

**Issue by Katlum** [Naltox/telegram-node-bot#203]
The link (http://nabovyan.xyz/telegram-node-bot/) is down. You can still access throw the internet archive wayback machine, but I think that I would be better if the docs here on github didn't list to a broken link.

Link for docs in wayback: https://web.archive.org/web/20180402035304/http://nabovyan.xyz/telegram-node-bot/#toc18__anchor

**Comment by rotimi-best** [comment]
I agree with you. I have been trying to get to the main website to see me documentations but I noticed it was broken. Thanks for the wayback link )
